spring-cloud-starter-stream-app-function/README.adoc

```diff
@@ -0,0 +1,56 @@
+//tag::ref-doc[]
+= Spring Cloud Function Bridge
+
+This application can be used to turn any plain jar containing a `Supplier`, `Function` or `Consumer`
+into a Spring Cloud Stream `Source`, `Processor` or `Sink` respectively.
+
+== Input
+
+Depends on the actual concrete type of the functional class used at runtime.
+
+== Output
+
+Depends on the actual concrete type of the functional class used at runtime.
+
+== Options
+
+The **$$function-app$$** application has the following options:
+
+$$function.location$$:: $$Location(s) of jar archives containing the supplier/function/consumer class to run.$$ *($$String[]$$, default: `$$[]$$`)*
+$$function.className$$:: $$The fully qualified class name of the supplier/function/consumer to run.$$ *($$String$$, default: `$$<none>$$`)*
+
+//end::ref-doc[]
+
+== Build
+
+```
+$ ./mvnw clean install -PgenerateApps
+$ cd apps
+
+You can find the corresponding binder based projects here. You can then cd into one one of the folders and build it:
+
+$ ./mvnw clean package
```

Comment:
I don't have a `mvnw` file in the apps directory

Reply:
Hmmm. Other starters end up having all the necessary files. Maybe there is something we need to do to get them copied? cc @sobychacko

Reply:
@ericbottard @trisberg we don't provide an `mvnw` in the `apps` directory. You need to `cd` into one of the generated apps (`function-app-kafka-10` for ex) and there you will find `mvnw`